tests/integration_test/run_int_test_locally.sh

```diff
@@ -19,21 +19,21 @@ vault_token_path=$2
 #use_caas=${14}
 #use_hmac=${15}
 
-script_dir=$repo_root/tests/integration_test
+SCRIPT_DIR="${repo_root}/tests/integration_test"
 
-bash $script_dir/integration_test.sh \
+bash ${SCRIPT_DIR}/integration_test.sh \
         "test" \
         "github" \
-        "master" \
+        "ra_update_to_caas_prod" \
         "github" \
-        "master" \
+        "rex-rhian-testing" \
```

Comment:
I think we have to leave this line as `rex-rhian-testing` until we fully migrate to caas-prod.